pkg/cli/cmds/agent_service_windows.go

```diff
@@ -0,0 +1,138 @@
+// +build windows
+
+package cmds
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+	"unsafe"
+
+	"github.com/rancher/k3s/pkg/cli/cmds"
+	"github.com/urfave/cli"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const (
+	defaultServiceDisplayName = "rke2"
```

Comment:
I think this is already defined in the version package in k3s and set at build time.

Reply:
it is indeed, version.Program, i've moved to this now